src/components/AddForm.vue

```diff
@@ -131,7 +131,7 @@
                   v-if="!phoneLayout || showMore"
                 >
                   <v-checkbox
-                    prepend-icon="mdi-progress-check"
+                    prepend-icon="mdi-data-matrix-scan"
```

Comment:
reject

Reply:
What about these?


`barcode-off` |  `fingerprint-off` | `eye-off`
:-------------------------:|:-------------------------:|:---------------------
<img width="90" alt="image" src="https://user-images.githubusercontent.com/19761269/96711145-b8c9a680-13ba-11eb-8ba5-95ccfcff433e.png">  |  <img width="90" alt="image" src="https://user-images.githubusercontent.com/19761269/96711166-c3843b80-13ba-11eb-9c61-92da4a236317.png">| <img width="90" alt="image" src="https://user-images.githubusercontent.com/19761269/96711236-dc8cec80-13ba-11eb-8dbb-edfef7d017d7.png">

Reply:
`barcode-off` seems ok, but `progress-check` is better.

Reply:
I think it needs to be an `off` icon because its function is in contrary to normal check boxes.

Reply:
I've gone with `barcode-off` in 50388a7 to better reflect the check box.